src/tests.cpp

```diff
@@ -207,8 +209,9 @@ void fq_codel(string good_examples_file, string bad_examples_file) {
     // Base Workload
     Workload wl(in_queue_cnt * 5, in_queue_cnt, total_time);
     for (unsigned int q = 0; q < last_queue; q++) {
+        Comp* comp_spec = new Comp(Indiv(metric_t::CENQ, q), op_t::GE, Time(1));
         wl.add_spec(
-            TimedSpec(Comp(Indiv(metric_t::CENQ, q), op_t::GE, Time(1)), total_time, total_time));
+            TimedSpec(comp_spec, total_time, total_time));
```

Comment:
Just add a new, no need for a separate variable.

Reply:
[Tests handle pointers more elegantly](https://github.com/all-things-networking/fperf/pull/27/commits/b9bc7fb16bff8b1a55e3f11963048e071ca1ad0b)